### ISSUE TITLE
Request for package boxedminipage

### ISSUE THREAD

**Issue by teepeemm** [brucemiller/LaTeXML#2110]
The package `boxedminipage` creates the environment `boxedminipage`, which is (drumroll) a boxed minipage.  (Slightly narrow and shorter than an `\fbox` around a minipage, so that the outer dimensions match the provided arguments).  It would be easy enough to copy the code from `LaTeX.pool.ltxml`, and add an additional class so that we could add the styling to put a border around it.  But it seems like it would be better to tell LaTeXML: "use the code for minipage, but also add this class".  Is there a good way to do that?

**Comment by tkw1536** [comment]
You should be able to just have latexml read the raw sty file using the [`--includestyles` option](https://math.nist.gov/~BMiller/LaTeXML/manual/commands/latexml.html#A1.S0.I1.ix4).
Something like:

```
latexmlc --includestyles example.tex --destination=example.xhtml
```

The following LaTeX:

```latex
\documentclass[a4paper]{article}
\usepackage{boxedminipage}
\begin{document}
\begin{boxedminipage}{0.5\textwidth}
    This is boxed.
\end{boxedminipage}
\end{document}
```

gives me (cutting off header and footer):

```xhtml
<div id="p1" class="ltx_para">
<p class="ltx_p"><span class="ltx_inline-para ltx_minipage ltx_align_middle ltx_framed_rectangle" style="width:210.0pt;">
<span id="p1.p1" class="ltx_para">
<span class="ltx_p">This is boxed.</span>
</span></span></p>
</div>
```

Notice how it adds both the classes `ltx_minipage` and `ltx_framed_rectangle`. 

In the browser this looks correct:

<img width="310" alt="image" src="https://github.com/brucemiller/LaTeXML/assets/2923242/9a1d9b92-6441-4daf-9872-b6ae60fd3a61">

**Comment by brucemiller** [comment]
fixed by #2132
